tests/rbd/rbd_flatten_image_feature_disable.py

```diff
@@ -39,6 +40,23 @@ def flatten_image_feature_disable(rbd, pool_type, **kw):
             log.error(f"Creation of clone {clone} failed")
             return 1
 
+        # set meta data value on clone
+        rbd.image_meta(
+            action="set",
+            image_spec=image_spec,
+            key="rbd_op_thread_timeout",
+            value=value,
+        )
+
+        # verify meta data value is set on clone image
+        if value != int(
```

Comment:
We are just validating `get` and `set` for `rbd_op_thread_timeout` can we cover actual functionlity level according to the value we set to `rbd_op_thread_timeout` ?

Reply:
@sunilangadi2 we are not validating the functionality for Block device general option, scenario is to verify if able to set the meta data value for clone images.